packages/algob/src/internal/deployer.ts

```diff
@@ -674,13 +676,13 @@ export class DeployerDeployMode extends DeployerBasicMode implements Deployer {
    * @param flags    - Deployments flags (as per SPEC)
    * @param payFlags - as per SPEC
    */
-  async fundLsigByName (
+  async fundLsig (
```

Comment:
need to update doc:
+ works both with name and filename
+ passed -> defined

Reply:
want to explain something here:

`this.getLsig` will basically return a logic signature **stored in a checkpoint**. Now, this could be a filename or a lsigname(as defined by user). So fileName essentially is just let's say a default lsig name.

`fundLsigByFile` actually takes the filename. The difference is it will "compile" the contract on the go (and won't load any data from checkpoint). Same with `loadLogicByFile`